apple/Sources/FerrostarMapLibreUI/Models/NavigationMapViewContentInsetMode.swift

```diff
@@ -12,7 +12,7 @@ public enum NavigationMapViewContentInsetMode {
     /// where the user location should appear toward the bottom of the map.
     ///
     /// This is used to accommodate a top InstructionView
-    case portrait(within: GeometryProxy, verticalPct: CGFloat = 0.75)
+    case portrait(within: GeometryProxy, verticalPct: CGFloat = 0.75, minVertical: CGFloat = 210)
```

Comment:
75% is too much for a tiny baby screen like my iPhone SE. Here's what it looks like with 7f49ceb reverted:

<img width="430" alt="Screenshot 2024-09-18 at 20 48 59" src="https://github.com/user-attachments/assets/87d48d8f-fcf4-4275-8e0a-965b8dab170f">

Note how the puck is almost underneath the ArrivalView. This is exacerbated by the bottom padding that this PR adds to the ArrivalView.

Reply:
**Puck Position**

On android we actually changed this value to 60% for that reason. We also adjusted the zoom based on travel activity. Both of these factors helped quite a lot with the general viewport. See: [NavigationCamera.kt](https://github.com/stadiamaps/ferrostar/blob/5a90b977cd572595f45ac8a1325f7579486aeca5/android/maplibreui/src/main/java/com/stadiamaps/ferrostar/maplibreui/runtime/NavigationCamera.kt). 

**Padding**

My preference when it comes to padding is to shoot for 16 as this is roughly Apple's iOS default value provided (e.g. if we just used `.padding(.vertical)` it would be 16 or close to it). With modern screen sizes and DPI's I tend to lean toward this default as it encourages slightly larger surfaces.

That said, there will always be different opinions and priorities with styling. Per that, I'd be all for making an "outer" padding part of the theme configuration. Thoughts @ianthetechie?

Reply:
Yeah, we had a lot of tweaking to get the puck positioning to an acceptable point on Android. We've done a lot more layout work there (ironic since both of us are very much iOS heavy haha), but the fragmentation is significantly more obvious over there.

This looks tolerable to me on a wide range of phones in testing. Tablets, the puck is maybe a bit high, but it's fine for now. We can always adjust later.

Making it part of the theme makes sense. Just to clarify, I think you're referring to the `NavigationUITheme`, right? In fact, moving it there would theoretically let us make the default implementation dynamic based on device type (either a size heuristic or idiom tablet checks). 

Assuming we're all agreed on that direction, it just struck me REALLY funny that this is a class and not a struct 🤔 Any reason not to make `NavigationUITheme` a struct @Archdoog?

Reply:
I've added some documentation and a rename to hopefully clarify this new parameter. This will conflict with a typo also fixed in #260.

Let me know if there's more you'd like me to change. I readily admit that I did this without consulting the Android codebase, so I appreciate the context. Let me know if you'd like me to do anything in particular to converge the implementations.